services/core-api/app/api/mines/permits/permit_extraction/tasks.py

```diff
@@ -44,13 +43,17 @@ def poll_update_permit_extraction_status(permit_extraction_task_id):
     Poll the permit conditions service for the status of the extraction task every 10s
     until the task is complete (SUCCESS or FAILURE).
     """
-    task = PermitSearchService().update_task_status(permit_extraction_task_id)
+    task, task_status = PermitSearchService().update_task_status(permit_extraction_task_id)
```

Comment:
I noticed that sometimes the frontend would display partial results, the reason was tracked down to the task status actually being updated before permit conditions were actually created